src/index.js

```diff
@@ -22,7 +23,21 @@ exports.start = (options, callback) => {
 
   const http = new Hapi.Server(config.hapi.options)
 
-  http.connection({ port, host })
+  let tls
+  if (options.key && options.cert) {
+    tls = {
+      key: readFileSync(options.key),
```

Comment:
`PEM allows the option of private keys being encrypted. Encrypted keys will be decrypted with options.passphrase` - nodeJS Docs

Seems like it would make sense to add the passphrase here as well

Reply:
Thanks for the info, I've added the option to that set of TLS settings